docs/integration/os/buildroot/quick_setup.rst

```diff
@@ -0,0 +1,55 @@
+Quick Setup 
+###########
+
+A Git repository is available that includes everything needed to test the 
+Buildroot setup without following the guide. It is intended for testing 
+purposes and can be used if you are already familiar with Buildroot concepts 
+and architecture.
+
+Get the repository
+
+.. code-block:: bash
+
+    git clone --recurse-submodules https://github.com/EDGEMTech/lvgl_buildroot_example.git
```

Comment:
It should be hosted in the LVGL organization. We can create an `lv_buildroot` repo.

Reply:
I agree, way better to have it in LVGL organization.